azure-pipelines.yml

```diff
@@ -0,0 +1,322 @@
+variables:
+- group: sonarsource-build-variables
+- name: BuildParameters.solution
+  value: SonarLint.VisualStudio.Integration.sln
+name: $(Build.BuildId)
+trigger:
+  branches:
+    include:
+    - master
+    - dev/*
+    - feature/*
+    - branch-*
+  batch: True
+jobs:
+- job: Phase_1
+  strategy:
+    maxParallel: 2
+    matrix:
+      2019:
+        vsTargetVersion: 2019
+      2022:
+        vsTargetVersion: 2022      
+  displayName: Phase 1
+  cancelTimeoutInMinutes: 1
+  pool:
+    vmImage: windows-2022
+  steps:
+  - checkout: self
+    submodules: recursive
+    fetchTags: false
+    persistCredentials: True
+  - task: PowerShell@2
+    displayName: Calculate if should sign Vsix
+    continueOnError: True
+    inputs:
+      targetType: inline
+      script: >-
```

Comment:
It pretty messy to have the scripts inline in one file. Can we load it from disk and commit to the repo these scripts?